client/src/main.jsx

```diff
@@ -35,7 +35,7 @@ Sentry.init({
 
 ReactDOM.render(
    
-    <React.StrictMode>
+    <>
```

Comment:
we should probably check this change with nicolas, and see if it is likely to have other side effects

Reply:
I think it is also fine if we keep React.StrictMode. What this does, to my knowledge, is that in dev mode load each component twice to ensure that it loads properly. In production I don't think this has any effect. Previously, I had some issues with eyeson connecting twice because of this, but since now I run "startSession()" conditionally on no existing video stream, it doesn't make a difference.